content/documentation/coffeeEditorNg/content.md

```diff
@@ -1,35 +1,45 @@
 </div>
 
-The core of the Coffee Editor NG is a model language contribution to the Model Hub.
-With this language contribution, we register the modeling language alongside the capabilities for handling this modeling language in the Model Hub.
-The contributed Coffee Editor NG Model is defined with Langium, an open-source language toolkit for textual languages.
-The Langium-based language is then integrated into the Model Hub API with the generic *EMF Cloud Model Hub Langium* integration library, so that Model Hub clients can interact with the language on model level and benefit from Langium's efficient persistence, cross-reference management and validation mechanism.
+## Model Hub Approach
 
-In order to also simplify the editing capabilities, including undo and redo, on model level -- rather than on text level -- we use the *EMF Cloud Editing Domain*, which provies state management, as well as a command API and a command stack for arbitrary JSON models.
+This approach focuses on comprehensive model management. It is built around a central Model Hub that offers robust capabilities such as command-based editing, state management with undo/redo, and persistence. This is ideal for scenarios where a powerful environment for managing complex models is required. Detailed examples for this approach can be found in the Model Hub repository:
 
-Finally, the Coffee Editor NG language contribution adds a custom coffee-model-specific API to be used by clients in order to provide reusable model queries and manipulation functions to the clients.
+https://github.com/eclipse-emfcloud/modelhub/tree/main/examples/theia
 
-<!--
-➡️ TODO Link to try now
 
-➡️ TODO Link to source code and a good entry point to look at the language contribution
--->
+## Langium-Based DSL Approach
 
-We will provide a live demo as well as a link to GitHub as soon as the code is published.
+This approach is designed for applications that require dedicated DSL support. Leveraging Langium, it provides advanced textual language features including parsing, reference resolution, and language server integration. This path is ideal for scenarios where the development of a domain-specific language is crucial. A prominent example demonstrating this approach is the CrossModel application, available at:
 
-In the remainder of this documentation, we use the Coffee Editor NG as an example to demonstrate how certain capabilities can be customized, extended or exchanged with other implementations.
-Please use the table of contents on the left to navigate to the respective topic of interest.
+https://github.com/CrossBreezeNL/crossmodel
+
+Both approaches are integral parts of EMF Cloud, offering distinct solutions tailored to different modeling requirements. Please use the table of contents on the left to navigate to the respective topics where you can find detailed guidance on implementation, customization, and best practices.
```

Comment:
```suggestion
Most modeling tools can be built with either of the two approaches, however, many modeling tools have rather a text-file-oriented model management perspective or a model-oriented model management perspective, at least they tend in one or the other direction. Tending in the one or the other direction makes certain implementation strategies a more natural fit.

Instead of enforcing one approach for all modeling tools, EMF Cloud embraces these differences and provides a set of different components for both implementation strategies. Please use the table of contents on the left to navigate to the respective topics where you can find detailed guidance on implementation, customization, and best practices.
```

I'd put this paragraph in the beginning of the next section.